server/api/routers/rag/router.py

```diff
@@ -105,6 +106,30 @@ async def get_rag_health(
     return await handle_rag_health(project_obj.config, str(project_dir), database)
 
 
+@router.get("/stats", response_model=RAGStatsResponse)
```

Comment:
Same thought/question here on whether or not we should use a path variable for database rather than query param

Reply:
Thought about it, but since you could query all DBs at once, I decided to leave this one at the root.